src/main/java/org/sonarqube/gradle/SonarQubePlugin.java

```diff
@@ -168,9 +179,17 @@ private void addGradleDefaults(final Project project, final Map<String, Object>
     configureForJava(project, properties);
     configureForGroovy(project, properties);
 
+    if (isAndroidProject(project)) {
+      AndroidUtils.configureForAndroid(project, properties);
+    }
+
     properties.putIfAbsent(SONAR_SOURCES_PROP, "");
   }
 
+  private static boolean isAndroidProject(Project project) {
+    return project.getPlugins().hasPlugin("com.android.application") || project.getPlugins().hasPlugin("com.android.library")|| project.getPlugins().hasPlugin("com.android.test");
```

Comment:
I hope so ;) This was suggested by Gradle guys: https://discuss.gradle.org/t/improve-sonarqube-plugin-to-support-android-projects/12368/2?u=henryju